__tests__/redux/actions/getArticles.test.js

```diff
@@ -76,7 +91,57 @@ describe('Should get article', () => {
     });
   });
 });
-describe('Should get profile', () => {
+describe('Should create article', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+    store.clearActions();
+  });
+  test('should get user profile', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 201,
+        response: { article: {} },
+      });
+    });
+
+    return store.dispatch(createArticle({
+      title: 's', description: 's', body: 's', tagList: 's',
+    }))
+      .then(() => {
+        expect(store.getActions().length).toEqual(1);
+      });
+  });
+});
+describe('Should Update article', () => {
```

Comment:
@rukundoeric provide the name of the component under test in the describe ('........')